packages/valory/skills/hello_world_abci/skill.yaml

```diff
@@ -97,6 +97,8 @@ models:
           version: {}
         voting_power: '10'
       hello_world_message: HELLO_WORLD!
+      owner_address: OWNER_ADDRESS
+      # TODO: Question: Are all these params necessary? Don't seem to be used.
```

Comment:
They might be used by other parts of the agent, like the Abci connection to sync all agents of a service.